src/JsonDB.ts

```diff
@@ -413,4 +413,41 @@ export class JsonDB {
             throw new DatabaseError("Can't save the database", 2, err)
         }
     }
+
+
+    /**
+     * Convert a router style path to a normal path
+     * @param path router based path to a correct base path
+     */
+    public async toPath(path: string ): Promise<string> {
```

Comment:
Hey, can you give me the run down of the algorithm ?

Does it try to match any property of the object to give the data path ?

Reply:
Hello Are you okay? I hope so.
The main idea is to simplify access to an object's path using the route style by always searching for the "id" property of objects within a list.

Reply:
I don't know if I knew how to explain it well, but every time I need to get the path of an item more deeply I have to use getIndex more and more.
So with this method I can simplify searches for deletion and update.

Reply:
Sorry I took so long to come back to you. (Dealing with a newborn at home ^^).

Only suggestion, could be to have a second parameter that tell the method what property to use (in case people don't want `id` but `_id` or something else).

Other than that, it's good to go for me.

Reply:
Firstly, congratulations on your baby. Ok, I'll adjust and resubmit the PR.